packages/client/components/ScopingSearchInput.tsx

```diff
@@ -38,7 +38,7 @@ interface Props {
   queryString: string
   meetingId: string
   linkedRecordName: string
-  service: TaskServiceEnum
+  service: IntegrationProviderServiceEnum
```

Comment:
-1 This seems unrelated and changes the enum to also include mattermost and msTeams which are not task integrations